Automattic/wp-calypso
# Site options: do not generate title and autofocus the input

## Proposed Changes

During the CfT, we received negative feedback about autogenerating titles. It's an important feature of a hosted site because the domain is derived from the site name, and we provide paid domain suggestions based on it, so that's something that the user needs to input.

[Another option](https://github.com/Automattic/wp-calypso/compare/hosting/do-not-add-suggestion-new-hosted-site...hosting/do-not-enforce-suggestion?expand=1) would be to not add anything but still let them pick a suggestion:

![image](https://github.com/Automattic/wp-calypso/assets/26530524/011877c4-849e-4662-a52c-f45f1652d660)

@javierarce how do you imagine the right adornment? Is this version OK? For small screens, what icon do you recommend?

## Testing Instructions

Pick any plan in `/setup/new-hosted-site` and verify that the input is empty and focused on mount.

![image](https://github.com/Automattic/wp-calypso/assets/26530524/82cc33bd-582f-4790-94e1-12226456422e)
